.iex.exs

```diff
@@ -0,0 +1,4 @@
+import_file_if_available("~/iex.exs")
```

Comment:
I think IEx will already load this file by default.

https://hexdocs.pm/iex/IEx.html#module-the-iex-exs-file

Reply:
Hmm, nevermind, it actually picks the first between the local and the global one, so if we need both we need this import 👍